tools/yulPhaser/Population.cpp

```diff
@@ -179,3 +177,11 @@ vector<Individual> Population::chromosomesToIndividuals(
 
 	return individuals;
 }
+
+vector<Individual> Population::sortIndividuals(vector<Individual> _individuals)
```

Comment:
```suggestion
vector<Individual> Population::soredIndividuals(vector<Individual> _individuals)
```
otherwise it could be seen as operating on a non-const ref.

Reply:
Good point. Renamed.